app/views/posts/show.html.erb

```diff
@@ -2,17 +2,29 @@
 <html>
 <head>
   <title>Post Details</title>
+
 </head>
 <body>
   <h1>Post Details</h1>
   
   <h2><%= @post.title %></h2>
   <p><strong>Author:</strong> <%= @post.author.name %></p>
-  
+  <p><strong>Likes:</strong> <%= @post.likes.count %></p>
```

Comment:
- Remove the user's name from the url upon navigating here. This was already pointed out by the previous reviewer.